runtime/compiler/runtime/SignalHandler.c

```diff
@@ -920,7 +920,7 @@ UDATA jit390Handler(J9VMThread* vmThread, U_32 sigType, void* sigInfo)
       const char* infoName;
       UDATA *controlPC;
       UDATA controlPCValue;
-      UDATA *controlFPC;
+      U_32 *controlFPC;
```

Comment:
While we're in this area of code we can remove this variable all together. All we care about is the value of the `DXC` on line 948. We can fold `infoValue` directly into it and remove the `controlFPC` variable.

Reply:
Sure, will do.

Reply:
Done.